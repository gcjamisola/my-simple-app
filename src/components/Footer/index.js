import React, { Component } from 'react';

import {StyledFooter} from '../../styles';

class Footer extends Component {
	render() {
		return (
			<StyledFooter>
				Copyright 2019
			</StyledFooter>
		);
	}
}

export default Footer;