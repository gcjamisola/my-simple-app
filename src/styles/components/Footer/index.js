import styled from 'styled-components';

const StyledFooter = styled.div`
	background: #000000;
	color: #f59500;
	height: 50px;
	display: flex;
	justify-content: center;
	align-items: center;
`;

export { StyledFooter };