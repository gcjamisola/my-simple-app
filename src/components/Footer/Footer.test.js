import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from './index';

describe('Footer component', () => {
	it('should render the component', () => {
		const component = shallow(<Footer />);
		const tree = toJson(component);
		expect(tree).toMatchSnapshot();
	});
});