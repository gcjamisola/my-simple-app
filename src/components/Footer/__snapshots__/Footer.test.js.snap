// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render the component 1`] = `
<styled.div>
  Copyright 2018
</styled.div>
`;
